substrate-node/tests/integration_tests.robot

```diff
@@ -439,52 +439,6 @@ Test Billing
 
     Tear Down Multi Node Network
 
-Test Solution Provider
-    [Documentation]    Testing creating and validating a solution provider
-    Setup Multi Node Network    log_name=test_create_approve_solution_provider    amt=${2}
```

Comment:
Maybe we could keep this test by just removing the billing part since there is no official 'green light' for completely getting rid of solution provider